bin/root-version.php

```diff
@@ -34,23 +35,11 @@ function get_last_tag_name(): string
 
     if (JSON_ERROR_NONE !== json_last_error()) {
         // TODO: switch to safe json parsing in the future
-        throw new RuntimeException(
```

Comment:
This is one is not desired IMO. I think it's inherited from the Symfony style, but IMO it only hurts readability here

Reply:
@theofidry, thanks for your reply.

Actually, I also agree with you, do you have any suggestion to let Style CI analyze the coding style successfully if revering these style changes?

Reply:
Should be configurable with `.styleci.yml`, as to what parameter exactly I do not know

Reply:
@theofidry, thanks for your reply.

I've disabled the `single_line_throw` setting on `.styleci.yml` setting to ignore this.